src/components/pages/Offer/StocksManager/StocksManagerContainer.js

```diff
@@ -10,29 +10,19 @@ import { selectProviderById } from '../../../../selectors/data/providersSelector
 import { selectOfferById } from '../../../../selectors/data/offersSelectors'
 
 export const mapStateToProps = (state, ownProps) => {
-  const {
-    match: {
-      params: { offerId },
-    },
-  } = ownProps
-
+  const { offerId } = ownProps
   const offer = selectOfferById(state, offerId)
-
-  if (!offer) {
-    return {}
-  }
-
   const product = selectProductById(state, get(offer, 'productId'))
   const stocks = selectStocksByOfferId(state, offerId)
-  const creationOfSecondStockIsPrevented = offer.isThing && stocks.length > 0
+  const isCreationOfSecondStockPrevented = offer.isThing && stocks.length > 0
```

Comment:
Est-ce qu'on ne pourrait pas inverser la logique partout ?
Si je comprend bien, la création de stock est limité à 1 quand c'est une Thing ?

```suggestion
  const isStockCreationAllowed = !(offer.isThing && stocks.length > 0)
```

L'approche _négative_ m'a fait des noeuds au cerveau ahah